### PR TITLE
Niloofar Sadeghi / Change deriv-go url structure

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by ShuvoHabib — APPROVED** [review]
LGTM

**Review by habib-deriv — DISMISSED** [review]

**Review by alamin-deriv — DISMISSED** [review]

**Review by sean-binary — CHANGES_REQUESTED** [review]
We will need to add the NGINX redirect in the deriv.com.conf file. You can follow the structure for `story`.

**Comment by niloo-fs** [comment]
> NGINX

I made the changes. would u pls check it again

File: deriv.com.conf [hunk 1]
```diff
@@ -41,6 +41,14 @@ server {
...
     }
 
+    location ~* "^/([\w]{2}|zh-tw|zh-cn)/deriv-go" {
+        return 301 https://$http_host/$1/deriv-go/;
+    }
+
+    location /deriv-go {
+        return 301 https://$http_host/deriv-go/;
+    }
+
```
Comment (sean-binary) [inline deriv.com.conf:51]:
```suggestion
    location ~* "^/([\w]{2}|zh-tw|zh-cn)/landing/deriv-go" {
        return 301 https://$http_host/$1/deriv-go/;
    }

    location /landing/deriv-go {
        return 301 https://$http_host/deriv-go/;
    }

```

Comment (sean-binary) [reply deriv.com.conf:51]: The original link should have `landing/` it in @niloo-fs .

**Review by sean-binary — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**